webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/General.js

```diff
@@ -97,6 +106,8 @@ General.propTypes = {
   handleInsecure: PropTypes.func.isRequired,
   handleInvalidField: PropTypes.func.isRequired,
   isLoading: PropTypes.bool.isRequired,
+  downloadUtility: PropTypes.oneOf(['curl', 'wget']),
```

Comment:
I don't think there will any more supported utils, but we can reuse `DownloadUtilities` constant here, so if one is updated, this is autoupdated as well.